### ISSUE TITLE
Mapping gender from ANZCTR

### ISSUE THREAD

**Issue by pwalsh** [opentrials/opentrials#176]
- no data for gender: http://explorer.opentrials.net/trials/164f3571-8248-4943-9682-8ea97708e352
- source has a gender field (requires parsing the string): https://www.anzctr.org.au/Trial/Registration/TrialReview.aspx?id=365397&isReview=true

**Comment by vitorbaptista** [comment]
This is blocked waiting on the licensing letters from ANZCTR

**Comment by pwalsh** [comment]
WONTFIX, pending licensing permissions. Will reopen if we get license restrictions lifted.
